app/components/profile_picture/index.tsx

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
```

Comment:
do we need this component for the Channel Navbar ?

Reply:
Removed.